frontend/src/components/HtmlContent/HtmlContent.scss

```diff
@@ -3,6 +3,10 @@
 // (thus wee cannot rely on Tailwind for such content)
 
 .inner-html-content {
+	word-break: break-word;
```

Comment:
the `break-word` value for `word-break` is deprecated (https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) I would rather not use it

Reply:
- https://github.com/cloudflare/wildebeest/pull/312/files#diff-f60c9d33987f39feb8d717ac5ce16e7f40a3fe602aab7980bf90b6fa26c68992L27
- https://github.com/cloudflare/wildebeest/pull/312/files#diff-06793c1a2e0a839f3637b588f420e7788434134bc412f16d1f1384506ee20675L49

```tsx
<article class="p-4 border-t border-wildebeest-700 break-words sm:break-normal">
```

I was using `word-break: break-word;` because the `sm:break-normal` sometimes did not break lines well.

`sm:break-normal` is not used, so I would remove it.

Also. I will not change the scss, as I think I can achieve this with Tailwind alone.

Reply:
Awesome! yeah good call! thanks! :slightly_smiling_face: